### DISCUSSION TITLE
iso someone to add functions to inventory system

### DISCUSSION THREAD

**Discussion by jacksonban** [inventree/InvenTree#5954] [Q&A]
hey i am wanting to add some functions to my inventory system 

Body Serial Number tracking for after sale tracking of warranty or repairs

Any pictures and supporting documents can be attached to the order

‘Quick-view’ of status on any vehicle in the shop and where it is in the upfitting/refitting process

Vehicle Sales Module that tracks all in-house labor and material, as well as all sublet work

be able to create a "build order" for a vehicle 

a few more things we can discuss once started

**Comment by SchrodingersGat** [comment]
There are a number of terms you are using here which I assume are specific to your industry - I don't fully understand all of them!

As there are a large number of different things to discuss here I suggest that either:

- Open a separate issue for each one so it can be discussed in detail
- reach out to support at inventree dot org
